modules/theme/src/themes/default/elements/button.overrides

```diff
@@ -57,6 +57,13 @@
                 background: @downBackgroundColor;
             }
         }
+        &.with-no-border {
```

Comment:
Since we use `LinkButton` do we need this?

Reply:
Yes, we do need it. I have referenced it as an inline className

```suggestion
                            className={ "m-1 p-2 with-no-border orange" }
```